addons/event_sale/static/tests/tours/event_configurator_ui.js

```diff
@@ -53,6 +53,13 @@ registry.category("web_tour.tours").add("event_configurator_tour", {
             trigger: "body:not(:has(.modal))",
         },
         ...tourUtils.clickSomewhereElse(),
+        tourUtils.editLineMatching("Event Registration", "Standard"),
+        tourUtils.editConfiguration(),
+        {
+            trigger: '.btn-close',
+            run: "click",
+        },
+        ...tourUtils.clickSomewhereElse(),
```

Comment:
Wouldn't add a step for that, test should mostly serve to test bulk flows, this is too specific. And tour steps are relatively expensive compared to other asserts.